ui/src/literature/components/__tests__/AssignConferenceDrawer.test.jsx

```diff
@@ -1,66 +1,122 @@
 import React from 'react';
-import { shallow } from 'enzyme';
-import { Set } from 'immutable';
+import { render, screen, fireEvent } from '@testing-library/react';
+import { Set, fromJS } from 'immutable';
+import { Provider } from 'react-redux';
+import { MemoryRouter } from 'react-router-dom';
 
 import AssignConferencesDrawer from '../AssignConferencesDrawer';
+import { getStore } from '../../../fixtures/store';
 
 jest.mock('react-router-dom', () => ({
+  ...jest.requireActual('react-router-dom'),
   useParams: jest.fn().mockImplementation(() => ({
     id: 123,
   })),
 }));
 
 describe('AssignConferencesDrawer', () => {
+  beforeAll(() => {
+    const rootElement = document.createElement('div');
+    rootElement.setAttribute('id', 'root');
+    document.body.appendChild(rootElement);
```

Comment:
question: Why do we need to this logic ?

Reply:
drawer component (and carousel etc) from antd needs root (or sometimes body) component present in the dom to 'pin' to it in order to render properly (i mean this https://ant.design/components/drawer#api:~:text=false-,getContainer,mounted%20node%20and%20display%20window%20for%20Drawer,-HTMLElement%20%7C%20()%20%3D%3E%20HTMLElement%20%7C%20Selectors)

Reply:
Thanks for clarifying !